Documentation/ApiOverview/Events/Events/FrontendLogin/LoginConfirmedEvent.rst

```diff
@@ -10,6 +10,12 @@ LoginConfirmedEvent
 The PSR-14 event :php:`\TYPO3\CMS\FrontendLogin\Event\LoginConfirmedEvent` is
 triggered when a login was successful.
 
+..  versionchanged:: 14.0
+    This event is now correctly dispatched, when a
+    logout redirect is configured. Previously the `actionUri` was used for
+    redirects after logout in which case the `LogoutConfirmedEvent` was not
+    triggered on logout.
```

Comment:
Previously the now removed `actionUri` was used as target for the logout form action, in which case the `LogoutConfirmedEvent` was not triggered on logout.

Reply:
Done